packages/core/lib/commands/develop/run.js

```diff
@@ -27,9 +28,15 @@ module.exports = async options => {
   const config = Config.detect(options);
   const customConfig = config.networks.develop || {};
 
-  const { mnemonic, accounts, privateKeys } = mnemonicInfo.getAccountsInfo(
-    customConfig.accounts || 10
-  );
+  // Respect user's number of accounts choice to 0 in truffle config
+  const isZeroAccount =
```

Comment:
Small detail, but how would you feel about renaming this to `isZeroAccounts`? When I first see this I think of THE zero account.